src/google-recaptcha-provider.tsx

```diff
@@ -131,6 +138,12 @@ export class GoogleReCaptchaProvider extends React.Component<
     if (!!nonce) {
       js.nonce = nonce;
     }
+    if (!!defer) {
```

Comment:
I think it should be safe to do just:

```javascript
js.defer = !!defer;
```

and get rid of the `if` statement. Can you please try ?

Reply:
Done!